### PR TITLE
Hot deploy env var

### PR DESCRIPTION
**Pull request by jhadvig**
Updated  both versions + docs.

@bparees PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,7 +218,11 @@ of the application source.
 Hot deploy
 ---------------------
 
-Hot deploy works in this image out of the box.
+In order to immidiatelly pick up changes made in your application source code, you need to run your built image with the `OPCACHE_REVALIDATE_FREQ=0` environment variable passed to the [Docker](http://docker.io) `-e` run flag:
```
Comment (bparees) [inline README.md]: immediately

**Comment by bparees** [comment]
one nit and will need openshift-docs update also.

**Comment by jhadvig** [comment]
@bparees comment addressed

**Comment by jhadvig** [comment]
Related PR's:
- cakephp - https://github.com/openshift/cakephp-ex/pull/19
- os-docs  - https://github.com/openshift/openshift-docs/pull/1043

File: README.md [hunk 2]
```diff
@@ -197,6 +197,9 @@ The following environment variables set their equivalent property value in the o
 * **OPCACHE_MEMORY_CONSUMPTION**
   * The OPcache shared memory storage size
   * Default: 16M
+* **OPCACHE_REVALIDATE_FREQ**
+  * The How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.
```
Comment (bparees) [inline README.md]: s/The How/How/

**Comment by bparees** [comment]
sorry, one more.

**Comment by jhadvig** [comment]
@bparees updated and rebased. thanks for noticing :+1:

**Comment by bparees** [comment]
lgtm

**Pull request merged**